spec/services/reports/list_spec.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+RSpec.describe Reports::List do
+  describe ".call" do
+    subject(:result) { described_class.call(**dependencies) }
+
+    fab!(:current_user) { Fabricate(:admin) }
+
+    let(:dependencies) { { guardian: current_user.guardian } }
+
+    it "does not include the about or storage_stats reports" do
+      expect(result[:reports].map { |r| r[:type] }).not_to include("about", "storage_stats")
+    end
+
+    it "does not include any mobile versions of page_view reports" do
+      expect(
+        result[:reports].filter { |r| r[:type].starts_with?("page_view") }.map { |r| r[:type] },
```

Comment:
Since this rather long statement is used 3 times, we could probably extract it in a `let`.